css/openhab.css

```diff
@@ -24,13 +24,23 @@ li.search:hover {
   border: none !important; 
```

Comment:
Some lines before are duplicate width entries (I probably forgot the remove one ...).